app/models/hearing_result.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class HearingResult < ApplicationRecord
+  validates :hearing_id, presence: true
+  validates :response, presence: true
```

Comment:
By calling this `response` aren't we binding it to only the Search/Response endpoint? This model is also used for the hearing result updates pushed from the Common Platform without a search right? 
What about just calling it `hearing`?

Reply:
Both the [search](https://github.com/ministryofjustice/hmcts-common-platform-mock-api/blob/master/lib/schemas/api/results.hearingResultedResponse.json) and [publish](https://github.com/ministryofjustice/hmcts-common-platform-mock-api/blob/master/lib/schemas/api/hearing-resulted.json) responses have a `hearing` and `sharedTime`, hence I chose to call it `HearingResult` rather than just `Hearing`, but i'm happy to change it to `Hearing`
About `response`, in the context of a request/response rather than search/response. Would it make sense to have it called something like `body` (I've assumed we are storing the response body only for a successful response, should we be concerned with the response status, headers etc?)

Reply:
No problem with the `HearingResult` (as can be read as some sort of output from a hearing). I would be happy either way `Hearing` or `HearingResult`.

On the `response`  I like your `body` suggestion :clap:. Good name.
Was concerned about the `HearingResulted` coming from HMCCTS being stored here too and `response` seems out of context in that case. But `body` seems perfect! :)